reliability/core-it/src/androidTest/kotlin/com/datadog/android/core/integration/tests/InternalSdkCoreTest.kt

```diff
@@ -406,8 +408,8 @@ class InternalSdkCoreTest : MockServerTest() {
             .apply {
                 _InternalProxy.allowClearTextHttp(this)
             }
-            .setBatchSize(forge.getForgery())
-            .setUploadFrequency(forge.getForgery())
+            .setBatchSize(BatchSize.SMALL)
```

Comment:
probably it should be different here? Like big batches + `setBatchProcessingLevel(BatchProcessingLevel.HIGH)`, since we want to upload as much as possible as fast as possible.